Hardcodet.NotifyIcon.Wpf/Source/NotifyIconWpf/TaskbarIcon.cs

```diff
@@ -156,10 +150,20 @@ public TaskbarIcon()
         #endregion
 
         #region Custom Balloons
+        /// <summary>
+        /// A delegate to handle customer popup positions.
+        /// </summary>
         public delegate Point GetCustomPopupPosition();
 
-        public GetCustomPopupPosition CustomPopupPosition;
+        /// <summary>
+        /// Specify a custom popup position
+        /// </summary>
+        public GetCustomPopupPosition CustomPopupPosition { get; set; }
```

Comment:
@punker76 @Lakritzator  I assume I had a reason to not declare that as a property, but I haven't done Windows UI for quite a while. Are there any implications to this with regards to data binding?

Reply:
Do you mean, as long as it's a field it will not be available for Binding? Because currently nothing would be binding to it, which makes sense..

That should be something @punker76 know better.

Actually that would be the only reason I could come up with against changing it, in general it's bad habit to make fields publicly available. (I actually didn't really look for them)

Reply:
I'm not sure, I just have a (six year old ;) deja-vu. Plus I really dislike public fields myself, so there's a bit of a red flag for me. Let's see what Jan thinks about it before I go on a wild goose chase ;)

Reply:
Red flag... good one, I know this feeling, I have some office interop code for Greenshot where I somehow can't remove an assignment which from the compiler's point of view doesn't make any sense.

If it turns out this is something like that, a comment is needed and the discussion had value! :+1:

Reply:
Yup! In any case, this PR here is super helpful to me, too since my Win-Fu is quite rusty. Very happy with the conversations here, so thanks a lot for that :)

Reply:
I think `public GetCustomPopupPosition CustomPopupPosition { get; set; }` is good here, because the Type here is a delegate(action) which is normally set only once. So a binding makes IMO no sense.

Reply:
Alright then! If there's nothing you'd like to add, I'll merge this one now.
(we can adjust the build targets when we introduce the .NET Core support)